app/controllers/taxon_changes_controller.rb

```diff
@@ -404,7 +417,27 @@ def analyze_ids
       format.json { render json: response[:json], status: response[:status] }
     end
   end
+
+  def withdraw
+    @taxon_change = TaxonChange.find( params[:id] )
```

Comment:
It isn't necessary to look up the taxon change here or in `restore` below, as it's already defined by the `load_taxon_change` before_action that will be called for these two methods

Reply:
removed